reconcile/utils/terrascript_client.py

```diff
@@ -3071,11 +3071,11 @@ def populate_tf_resource_elasticsearch(self, resource, namespace_info):
                 ", and - (hyphen). " +
                 f"{values['identifier']}")
 
-        elasticsearch_version = values.get('elasticsearch_version', 7.7)
+        elasticsearch_version = values.get('elasticsearch_version')
```

Comment:
no default? in TF it's an optional parameter.
On the other hand, better to fail if you expect all apps-sre customers to specify it.

Reply:
We need to skip 
```
es_values["elasticsearch_version"] = elasticsearch_version
```
if elasticsearch_version is none.